### DISCUSSION TITLE
Multiple Prometheuses?

### DISCUSSION THREAD

**Discussion by ameddin73** [topolvm/pvc-autoresizer#200] [Q&A] [answered]
Great tool! I've got a cluster that supports multi-tenancy through multiple prometheus stacks. Is multiple prometheuses supported? If not explicitly, can I have multiple instances of pvc-autoresizer co-located and pointing to different prometheus urls?

**Comment by toshipp** [comment, marked as answer]
pvc-autoresizer does not support multiple prometheuses.
Although I don't know how you split your prometheus instances, I consider you need only one pvc-autoresizer instance that refers kubelet metrics  (see https://github.com/topolvm/pvc-autoresizer/blob/d4c843e4a1f5e649b0a544b4cb8f94dceb7f37b5/runners/metrics_client.go#L16-L19).
